packages/editor/src/components/block-list/block.js

```diff
@@ -655,8 +650,7 @@ const applyWithDispatch = withDispatch( ( dispatch, ownProps ) => {
 			selectBlock( clientId, initialPosition );
 		},
 		onInsertBlocks( blocks, index ) {
-			const { rootClientId, layout } = ownProps;
-			blocks = blocks.map( ( block ) => cloneBlock( block, { layout } ) );
```

Comment:
`cloneBlock` does more than just adding the layout, just want to ensure we don't break anything here. (For example, it clones the inner blocks too etc...)

Reply:
I'm fairly certain this was only used to inject the layout, but I'll double-check.

Reply:
The only thing which should change is the client ID, which is not something which should be relied upon anyways, and certainly not so for a block which is created but yet to be inserted.